ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/RedHat.yml

```
@@ -52,11 +51,11 @@
 ##########################
 # Additional build tools #
 ##########################
-- name: Install additional build tools if NOT RHEL 8
+- name: Install additional build tools if NOT RHEL8 and above 
```

Remove trailing whitespace
```suggestion
- name: Install additional build tools if NOT RHEL8 and above
```